packages/quantic/force-app/main/default/lwc/quanticFacetManager/e2e/pageObject.ts

```diff
@@ -0,0 +1,66 @@
+import type {Locator, Page} from '@playwright/test';
+import {
+  insightSearchRequestRegex,
+  searchRequestRegex,
+} from '../../../../../../playwright/utils/requests';
+
+export class FacetManagerObject {
+  constructor(public page: Page) {
+    this.page = page;
+  }
+
+  get facetManager(): Locator {
+    return this.page.locator('c-quantic-facet-manager');
+  }
+
+  get facetManagerItems(): Locator {
+    return this.facetManager.locator('.facet-manager__item');
+  }
+
+  getFacetManagerItemByIndex(index: number): Locator {
+    return this.facetManagerItems.nth(index);
+  }
+
+  /**
+   * Filters out the date_input facets because they are not displayed in the facet manager.
```

Comment:
that's not specific to the date_input,  so this comment needs to be updated.
This function filters any facetId id that contains `_input` which is a suffix added to any facet id when the facet in question contains a filter input.

Reply:
additionally this function does not seem to belong to this file, it's not related to the page object, it's more like a helper function only needed in the spec file, so I would put it there.